app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,43 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
+        <ui-select-match placeholder="Select a storage class">
+          <span>
+            {{$select.selected.metadata.name}}
+          </span>
+        </ui-select-match>
+        <ui-select-choices
+          repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } | orderBy: 'metadata.name'">
+          <div>{{ sclass.metadata.name }}
```

Comment:
> the match is highlighted with the code

I pulled down your changes, and it's not highlighting the matching text.

The truncation is because of this CSS style.

https://github.com/spadgett/origin-web-console/blob/f2c009c13249762921f94576a39b223d5fca3320/app/styles/_select.less#L44

@rhamilto Can we remove or change that style? It's caused problems more than once :/